playwright/test/exhibition-guides.test.ts

```diff
@@ -27,3 +43,152 @@ test('Redirects to another format if we have an EG preference and come from a QR
     /\/bsl[?]usingQRCode=true&stopId=witness#witness/
   );
 });
+
+test.describe('(2) | with egWork toggle: ', () => {
+  // TODO remove this when we stop supporting legacy QRs
```

Comment:
Perhaps if we created an issue for this in GH, we could link to that from here (and vice versa, probs using a code search)?

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/11131